widget/entry.go

```diff
@@ -497,6 +497,23 @@ func (e *Entry) pasteFromClipboard(clipboard fyne.Clipboard) {
 	Renderer(e).(*entryRenderer).moveCursor()
 }
 
+// selectAll selects all text in entry
+func (e *Entry) selectAll() {
+	e.Lock()
+	if e.selectKeyDown == false {
```

Comment:
Can you share what this does?
These lines are not tested and I don't understand the purpose

Reply:
This was to make sure selectAll isn't triggered when shift is held, but i think this is redundant anyway as such scenario wouldn't trigger selectAll. My bad.